src/main/java/Duke.java

```diff
@@ -6,5 +9,93 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("\t------------------------------------------");
+        System.out.println("\tHello there! I'm Duke.");
+        System.out.println("\tWhat can I help you with?");
+        System.out.println("\t__________________________________________\n");
+
+        ArrayList<Task> tasks = new ArrayList<>();
+        boolean runLoop = true;
+
+        while (runLoop) {
+
+            String userInput = sc.nextLine().trim();
+
+            if (userInput.equals("bye")) {
+                // exits the program
+                System.out.println("\t------------------------------------------");
+                System.out.println("\tSee you soon! Goodbye! ^.^");
+                System.out.println("\t__________________________________________\n");
+                runLoop = false;
+            } else if (userInput.equals("list")) {
+                System.out.println("\t------------------------------------------");
+                if (tasks.size() == 0) {
+                    // if list is empty
+                    System.out.println("\tYour list is empty!");
+                }
+                else {
+                    // displays the list of tasks
+                    System.out.println("\tHere are the tasks in your list: ");
+                    int i = 1;
+                    for (Task task : tasks) {
+                        System.out.println("\t" + i++ + "." + task.toString());
+                    }
+                }
+                System.out.println("\t__________________________________________\n");
+            } else if (userInput.split(" ")[0].equals("done")) {
+                // check if task exists
+                System.out.println("\t------------------------------------------");
+                // check if
+                int taskIndex = Integer.parseInt(userInput.split(" ")[1]) - 1;
+                if (taskIndex > tasks.size()-1) {
+                    System.out.println("\tTask " + taskIndex + " does not exist! Please try again.");
+                }
+                else {
+                    // sets a task as done
+                    System.out.println("\tGreat job! I've marked this task as done: ");
+                    Task task = tasks.get(taskIndex);
+                    task.markAsDone();
+                    System.out.println("\t" + task.toString());
+
+                }
+                System.out.println("\t__________________________________________\n");
+            } else {
+                // adds a task to the list
+                System.out.println("\t------------------------------------------");
```

Comment:
I like the usage of a different form of lines, one:----- and one––––––––, it performs different meanings and let users easily understood.